tests/kernel/mem_protect/userspace/README.rst

```diff
@@ -0,0 +1,155 @@
+.. _protection_tests:
+
+Protection tests
+#################################
+
+Overview
+********
+This test case verifies that protection is provided
+against the following:
+
+* Inheritance of permissions
+* Memory domain features.
+* k objects validations.
+
```

Comment:
Do we really need a README.rst at all for this?  I had one in protection because it started life as a sample rather than a test and the samples had them, but I don't see them being used in the tests.  Easier if it can just be dropped entirely.

Reply:
I agree, I don't think these are required any more. Test case output is more or less standardized now.